src/IK/tests/QPInverseKinematicsTest.cpp

```diff
@@ -96,7 +96,10 @@ InverseKinematicsAndTasks createIK(std::shared_ptr<IParametersHandler> handler,
     // prepare the parameters related to the size of the system
     const Eigen::VectorXd kpRegularization = Eigen::VectorXd::Ones(kinDyn->model().getNrOfDOFs());
     const Eigen::VectorXd weightRegularization = kpRegularization;
-    handler->getGroup("REGULARIZATION_TASK").lock()->setParameter("kp", kpRegularization);
+
+    auto regularizationTaskGroup = handler->getGroup("REGULARIZATION_TASK").lock()->clone();
+    regularizationTaskGroup->setParameter("kp", kpRegularization);
+    REQUIRE(handler->setGroup("REGULARIZATION_TASK", regularizationTaskGroup));
```

Comment:
I would prefer this change would have not been needed. With the not ``const`` method these modifications should not be necessary, right?